apps/froussard/func.yaml

```
@@ -0,0 +1,16 @@
+specVersion: 0.36.0
+name: froussard
+runtime: typescript
+registry: kalmyk.duckdns.org/lab
+created: 2025-03-30T10:30:06.936905-07:00
+build:
+  builderImages:
+    platforms: linux/arm64
+    s2i: registry.access.redhat.com/ubi8/nodejs-22-minimal
+  builder: s2i
+  buildEnvs:
+  - name: BP_NODE_RUN_SCRIPTS
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)